Resources/Prototypes/_NF/Guidebook/shipyard.yml

```diff
@@ -5,6 +5,7 @@
   priority: 4
   children:
   - ShipyardAmbition
+  - ShipyardAkupara
```

Comment:
Not a strict requirement, but try to keep it in alphabetical order

Reply:
Oops, off by one spot. Fixed.